src/components/CommunityEditionHead.tsx

```diff
@@ -0,0 +1,74 @@
+import { Locale } from 'i18n.config';
+import Head from 'next/head';
+import { ReactNode } from 'react';
+
+type Props = {
+  children?: ReactNode;
+  metadata: {
+    title: string;
+    description: string;
+    keywords: string[];
+    opengraph?: {
+      title: string;
+      description: string;
+      locale: Locale & {
+        alternate: Locale;
+      };
+      image: string & {
+        width: string;
+        height: string;
+        alt: string;
+      };
+    };
+    twitter?: {
+      type: string;
+      title: string;
+      description: string;
+      image: string & {
+        width: string;
+        height: string;
+        alt: string;
+      };
+    };
+  };
+};
+
+export default function CommunityEditionHead({ children, metadata }: Props) {
```

Comment:
maybe not the best name, I'd change with this
```suggestion
     export default function HeadComponent({ children, metadata }: Props) {
```